### PR TITLE
feat: add summary option in CLI

### PR DESCRIPTION
**Pull request by sfat**
implementation for #22 

 ## Normal report, without specifying the `-s` parameter
```cat test/fixtures/test-report.json | node dist/index.js  -o before.html```

detailed-report

 ## Summary report, providing `-s` parameter
```cat test/fixtures/test-report.json | node dist/index.js  -o before.html```

with-summary-only

#### One `fixedIn` value
on-fixed-in

#### Multiple `fixedIn` value
multiple-fixed-ins

### When no value for`vuln.fixedIn` present, fallback to a default message
fallback-message

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -1,38 +1,48 @@
 #!/usr/bin/env node
 
+import program = require('commander');
```
Comment (lili2311) [inline src/index.ts]: not seeing this in the package.json & lockfile above, please add

File: package.json [hunk 2]
```diff
@@ -32,7 +32,6 @@
   "snyk": true,
   "devDependencies": {
     "@types/marked": "^0.6.5",
-    "@types/minimist": "^1.2.0",
```
Comment (lili2311) [inline package.json]: how the types for minimist are dropped?

File: src/index.ts [hunk 3]
```diff
@@ -1,38 +1,48 @@
...
-  output = argv.o; // grab the next item
+if (program.output) { // output destination
+  output = program.output; // grab the next item
   if (typeof output === 'boolean') {
     output = undefined;
   }
 }
 
-SnykToHtml.run(source, template, onReportOutput);
+console.log('summary: ' + program.summary);
```
Comment (lili2311) [inline src/index.ts]: left over?

File: template/test-report.vuln-card.hbs [hunk 4]
```diff
@@ -70,80 +70,86 @@
...
-        {{else}}
-            <h3 class="card__section__title">Detailed paths</h3>
-        {{/if_eq}}
-
-        <ul class="card__meta__paths">
-            {{#each list}}
-                {{#isDoubleArray from}}
-                    {{#each .}}
-                    <li class="list-paths__item">
+        {{#unless showSummaryOnly}}
```
Comment (lili2311) [inline template/test-report.vuln-card.hbs]: could you add a screenshot with & without `-s` to the PR description please to help visualise

File: README.md [hunk 5]
```diff
@@ -15,6 +15,15 @@ First, Install the Snyk JSON to HTML Mapper using npm:
 
 Alternatively, you can skip this step, clone the repo and run the script locally (using `node ./snyk-to-html.js`)
 
+## Options of the CLI
+
+1. `-t` or `--template`-  Template location for generating the html. Defaults to template/test-report.hbs
+2. `-i` or `--input`   -  Input path from where to read the json. Defaults to stdin
+3. `-o` or `--output`  -  Output of the resulting HTML. Example: -o snyk.html. Defaults to stdout
+4. `-s` or `--summary` -  Generates an HTML with only the summary, instead of the details report Example: -o snyk.html. Defaults to stdout
```
Comment (orsagie) [inline README.md]: copypaste for example here is for -o instead of -s

File: README.md [hunk 6]
```diff
@@ -35,6 +44,13 @@ Change directory to your package's root folder, then use of the two ways below t
 
    `snyk-to-html -i results.json -o results.html`
 
+3. By default, details about each vulnerability is shown.
```
Comment (lili2311) [inline README.md]: ❤️

File: README.md [hunk 7]
```diff
@@ -15,6 +15,15 @@ First, Install the Snyk JSON to HTML Mapper using npm:
 
 Alternatively, you can skip this step, clone the repo and run the script locally (using `node ./snyk-to-html.js`)
 
+## Options of the CLI
```
Comment (lili2311) [inline README.md]: wonderful seeing this documented 🎉

File: template/test-report.vuln-card.hbs (hunk 4 above)
Comment (sfat) [reply template/test-report.vuln-card.hbs]: sure! description updated

File: src/index.ts (hunk 3 above)
Comment (sfat) [reply src/index.ts]: Yes

File: src/index.ts (hunk 1 above)
Comment (sfat) [reply src/index.ts]: updated

File: package.json (hunk 2 above)
Comment (sfat) [reply package.json]: I've also removed `minimist` all together as it is not being mantained for almost 5 years

**Pull request merged**